src/alpine/3.9/amd64/Dockerfile

```diff
@@ -20,6 +20,7 @@ RUN apk add --no-cache \
         libtool \
         libunwind-dev \
         linux-headers \
+        lld \
```

Comment:
Alpine 3.9 is EOL and will be deleted soon: https://github.com/dotnet/dotnet-buildtools-prereqs-docker/issues/442

Reply:
We still use it to build runtime for Alpine and we don't have equivalent one for 3.10 yet. Would you like me to add the 3.10 one in this PR?

Reply:
I mean the 3.11, as the 3.10 is EOL too.

Reply:
Alpine 3.11 is EOL in Nov 2021 so we'll just have to do it all over again then.  It'd be preferable to just jump to the latest, Alpine 3.13 to buy us an extra year.